src/accordion/Accordion.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @overview An Accordion module comprised of sections that toggle open and close
+ * @module Accordion.js
+*/
+
+export const Accordion = {
+
+  accordionSection : `.js-accordion-section`,
+  accordionSectionTrigger: `.js-accordion-section-toggle`,
+  accordionSectionContent: `.js-accordion-section-content`,
+  openClass: `is-open`,
+
+  /**
+   * Inits accordion elements
+   * @returns {void}
+  */
+  init() {
+    this.accordionSections  = document.querySelectorAll(this.accordionSection);
```

Comment:
This could just be a const if it's not being used outside of init(), then you won't have to use `this.`